nix/m1-support/u-boot/default.nix

```diff
@@ -21,20 +21,29 @@ in (buildPkgs.buildUBoot rec {
   extraMeta.platforms = [ "aarch64-linux" ];
   filesToInstall = [
     "u-boot-nodtb.bin.gz"
-    "m1n1-u-boot.macho"
     "m1n1-u-boot.bin"
   ];
   extraConfig = ''
     CONFIG_IDENT_STRING=" ${version}"
+    CONFIG_VIDEO_FONT_4X6=n
+    CONFIG_VIDEO_FONT_8X16=n
+    CONFIG_VIDEO_FONT_SUN12X22=n
+    CONFIG_VIDEO_FONT_TER12X24=n
```

Comment:
I do not understand why you add the patch `mps-u-boot-ter12x24.patch` if you do not enable `CONFIG_VIDEO_FONT_TER12X24`?

Reply:
Perhaps someone will prefer that font over the largest one I chose.  I don't think it affects code size, because based on the code I think the font inclusion is guarded by ifdefs based on the config options.